.github/workflows/run-tests.yml

```diff
@@ -69,22 +69,49 @@ jobs:
 
             - run: node_modules/.bin/webpack
 
+            - name: Install Fork CMS
+              run: cp src/Modules/Installer/tests/fork-cms-installation-configuration.yaml fork-cms-installation-configuration.yaml && bin/console forkcms:installer:install && sed -i 's/forkcms_test/forkcms/g' .env.local
+              env:
+                  FORK_DATABASE_USER: root
+                  FORK_DATABASE_PASSWORD: kingtriton
+
             - name: Execute tests with coverage
               if: ${{ matrix.php == '8.1' }}
-              run: bin/simple-phpunit --testsuite=${{ matrix.testsuite}} --coverage-clover=${{ matrix.testsuite}}.clover
+              run: bin/simple-phpunit --testsuite=${{ matrix.testsuite}} --coverage-clover=${{ matrix.testsuite}}.clover -v
+              env:
+                  FORK_INSTALLATION_LOCALE_ENV_PATH: .env.local
+                  APP_ENV: test
+                  FORK_DATABASE_HOST: 127.0.0.1
+                  FORK_DATABASE_PORT: 3306
+                  FORK_DATABASE_NAME: forkcms_test
+                  FORK_DATABASE_USER: root
+                  FORK_DATABASE_PASSWORD: kingtriton
 
             - name: Execute tests without coverage
               if: ${{ matrix.php != '8.1' }}
-              run: bin/simple-phpunit --testsuite=${{ matrix.testsuite}}
```

Comment:
Rebased the commit to remove the -v
The original test suite was already configured like this, I think it's mostly to win speed since once you have coverage of one you don't really need it of the other ones, unless you know of a reason why we should run the coverage in all versions of the matrix?

Reply:
nm, I did not spot the that it will only be done on PHP 8.1